calcom/cal.com
# Bugfix/opt in booking shown on attendee booking page

## What does this PR do?
Fixes issue when booking a time for an opt-in event type if the attendee is another cal.com user the booking with the meeting confirmation actions appears in the attendee booking page.

When a user enters the bookings page, bookings are queried on the criteria if the user is the organizer or if the user is an attendee. On the booking list item component the current logic to show the meeting confirmation actions were if the meeting was not confirmed and rejected.

The fix I implemented is when querying for bookings to include the organizer's user id. On the booking list item component I then added to the current logic to only show the meeting confirmation actions if the current user id matches the organizer user id.

Fixes #1982 

## Type of change




## How should this be tested?

- Log in as two separate users
    - I logged in as one user in a normal chrome window and as another user in an incognito window
- For user 1 create a opt-in event type
- Visit the booking link as user 2 and book a time
    - **The attendee email must match user 2's email**
- With user 2 visit the bookings page
    - The booking should not display the meeting confirmation actions
- With user 1 visit the bookings page
    - The booking should display the meeting confirmation actions
- With user 1 confirm the meeting
- For both users the cancel and reschedule actions should now appear
    - For user 2 you will have to refresh the page
